src/devices/Ssd1306/Command/SetPreChargePeriod.cs

```diff
@@ -0,0 +1,58 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+namespace Iot.Device.Ssd1306.Command
+{
+    public class SetPreChargePeriod : ICommand
+    {
+        /// <summary>
+        /// This command is used to set the duration of the pre-charge period.
+        /// The interval is counted in number of DCLK, where RESET equals 2 DCLKs.
+        /// </summary>
+        /// <param name="phase1Period">Phase 1 period with a range of 1-15.</param>
+        /// <param name="phase2Period">Phase 2 period with a range of 1-15.</param>
+        public SetPreChargePeriod(byte phase1Period = 0x02, byte phase2Period = 0x02)
+        {
+            if (phase1Period < 0x01 || phase1Period > 0x0F)
```

Comment:
These range checks can be made faster (with fewer total branches/jumps).
[sharplab.io](https://sharplab.io/#v2:EYLgHgbALANALiAhgZwLYB8ACAGABJgRgG4BYAKEwGZ8AmXAYVwG9zc39rgB7LgG1wBqiXgEsAJojgBTAgApgAT2m5gBGCqVSVNAJTNW7QyIBmueQVwAeXNjDYL6dCosA+G3YBiOg4bYsyvoH4AOy4xsLIUqQBQQC+Pr4mZsB01rb2uI7auG7pXgmG/kG+mKHhvJHRxfExgaW4cABOAK5RCTWGCVQqPPxCohLSNPKazuqKyil6RUFJsgCEAJIAdgBKiMsA5lLm6ulq7the3rW+M8Vs9eWVBewds6YLK+tbOyl7dgd5OifF5xdXCJtU6Ge51UJNVpVO4JBIAB0aIgAbpItIQID0+LhnhttrJmiJlnBcCjeK11ASibhkHBEI04BTCcSpMsxL92P9DPVZKTWrgALTU2n0vSWAC8ZhZYgFQrpcB00LYNViQA)

Use a helper like this:
```C#
private static bool InRange(uint value, uint start, uint end)
{
    return (value - start) <= (end - start);
}
```

Reply:
That's a nifty helper.  I will update where needed.  Thx